flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/agg/WindowAggregateTest.scala

```diff
@@ -127,6 +162,26 @@ class WindowAggregateTest(aggPhaseEnforcer: AggregatePhaseStrategy) extends Tabl
     util.verifyRelPlan(sql)
   }
 
+  @TestTemplate
+  def testTumble_OnProctimeWithCDCSource(): Unit = {
+    assumeThat(isTwoPhase).isTrue
```

Comment:
Is there any particular reason to just enable two-phase test?

Reply:
Because currently, we don't support two-phase optimization for window agg while using proctime. Although we only test two-stage, but the plan is same with the result of one-phase. Not testing one-phase may help to reduce the size of the xml.